CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# CDC (Change Data Capture) Change Log
+
+All notable changes to this project are documented in this file.
+
+## [4.0.0-beta.2] - 2020-03-13
+
+Initial release of the change data capture, providing following features in this release
+
+- Support capturing change data from TiKV since v4.0.0-beta.2 and change data sort
```

Comment:
What do you mean `change data sort` here?

Reply:
removed the content of `change data sort`, it is a technical detail.